### ISSUE TITLE
Cursor jumps to next page on the bottom following a search

### ISSUE THREAD

**Issue by t-xigit** [infokiller/web-search-navigator#84]
I'm using Chrome: Version 69.0.3497.100 (Official Build) (64-bit)
On Windows 10

Since the latest update of Chrome after pressing enter cursor jumps to the bottom.
When I am disabling the extension the behavior is normal.

![image](https://user-images.githubusercontent.com/15726945/46205481-06c96880-c321-11e8-8277-17756ce7d2ce.png)

**Comment by vcfvct** [comment]
Was having the same issue as above, then I re-installed the extension and now no `arrow` shows up any more, however i still can see the link focus are changing when `up/down` key is pressed. 

Any ideas?

**Comment by tedzhu** [comment]
Definitely only started happening recently, might not be related to Chrome update. Is it the HTML structure of Google search result somehow changed?

**Comment by gudata** [comment]
same here. Chromium. Version 69.0.3497.81 (Official Build) Built on Ubuntu , running on Ubuntu 18.04 (64-bit)

**Comment by infokiller** [comment]
Sorry for this issue guys.

**TLDR**: You need to disable the extension from the store and install if from a Github release (see [instructions below](short-term-workaround]).

## Long story

The root cause is a change in the google html structure which broke the extension logic for identifying the search results.
This has already been fixed in the code, but unfortunately, the extension has been temporarily removed from the Chrome Webstore (see #60 - waiting for Google to approve it again). This removal prevents me from pushing updates to the version of the extension in the Webstore.

## Short term workaround

The short term workaround is to disable the extension installed from the Chrome Webstore and install the extension from a Github release.

1. Download the latest release from https://github.com/infokiller/web-search-navigator/releases
1. Extract the zip file to any directory you'd like, though you need to make sure the directory is not deleted while you want to use the extension
1. Go to extensions page in Chrome (URL: `chrome://extensions`)
1. Enable developer mode if needed
1. Disable other existing instances of Google Search Navigator (from the store or from loading another unpacked version)
1. Click "Load Unpacked" and select the directory you extracted the release zip into

## Longer term solution

Once Google reapproves the extension and #60 is resolved, I will be able to push updates to the Chrome Webstore version, making it the preferred way to use the extension again.

**Comment by infokiller** [comment]
This is a duplicate of #78 but I'll keep it open for discovery

**Comment by maximillianus** [comment]
Correct me if I am wrong, but this extension was previously approved by Google? As far as I know it is not approved and it is not available anywhere in Chrome Webstore (actually I just check it and it is still not available there). Kindly let me know the link to the webstore apps if any since the link on the github main page is giving me 404 error. Anyway, thanks for the update on the fix. I am having this issue as well.

**Comment by infokiller** [comment]
Yes it was previously available, but removed because of copyright issues (see #60). See the updated [Chrome installation instructions](https://github.com/infokiller/web-search-navigator#chrome) for how to install it.
